components/audioinput/microphone/microphone.go

```diff
@@ -60,8 +60,17 @@ func newMicrophoneSource(conf *Config, logger logging.Logger) (audioinput.AudioS
 
 	debug := conf.Debug
 
+	var constraints mediadevices.MediaTrackConstraints
```

Comment:
Does this have a creator function? We could also initialise the empty variable `constraints := mediadevices.MediaTrackConstraints`

Reply:
Should be able to initialize this similar to the current (video) `DefaultConstraints`.